component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -1588,13 +1590,14 @@ protected boolean retryAuthenticationEnabled() {
      * @throws AuthenticationFailedException : In occasions of failing sending the email to the user.
      */
     protected void triggerEvent(String userName, String tenantDomain, String userStoreDomainName,
-                                String notificationEvent, String otpCode) throws AuthenticationFailedException {
+                                String notificationEvent, String otpCode, String email) throws AuthenticationFailedException {
```

Comment:
Can we rename the parameter to "String email" to "String sendToAddress"

Reply:
that is a choice. you can change it.